config/crd/bases/config.nri_balloonspolicies.yaml

```diff
@@ -513,6 +520,18 @@ spec:
                   type: string
                 description: Reserved (CPU) resources for kube-system namespace.
                 type: object
+              showContainersInNrt:
+                description: |-
+                  ShowContainersInNrt controls whether containers in balloons
```

Comment:
nit: To address the concerns raised by @fmuyassarov, shouldn't we add here a word of warning about how enabling this will affect the amount of data pushed to the CR / API server from nodes with a large amount of containers ?

Reply:
Then again, I guess considering how mild the constraints are when running with the default etcd configuration, maybe there is no strict need for such a warning...

Reply:
I think this is a valid concern anyway. At least better to give a hint that there could be issues, this does not come for free. If it's not the size of the message (due to hundreds of containers running on the node), then at least about the extra api-server + etcd traffic (a lively 1000+ node cluster)...

Thanks @klihub & @fmuyassarov on these notes!

Reply:
A sentence about potential costs added.